circle.yml

```diff
@@ -15,15 +16,16 @@ dependencies:
     - "~/.cabal"
 
   override:
-    - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
-    - cp -r /home/ubuntu/git-town /home/ubuntu/.go_workspace/src/github.com/Originate/gt
-    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && go install
-    - rm -rf /home/ubuntu/.go_workspace/src/github.com/Originate/gt
+    - mkdir -p "$(dirname "$GO_PROJECT")"
+    - if [[ ! -e "$GO_PROJECT" ]]; then ln -s $HOME/git-town "$GO_PROJECT"; fi
+    - cd $GO_PROJECT && go install && go get gopkg.in/alecthomas/gometalinter.v1
```

Comment:
if you want to be able to do that, you need the link to go the other direction, so you'd need to do something like:
```bash
rm -rf "$GO_PROJECT"
mkdir -p "$(dirname "$GO_PROJECT")"
mv "$HOME/git-town" "$GO_PROJECT"
ln -s "$GO_PROJECT" "$HOME/git-town"
```